ensemble_eeg/ensemble_edf.py

```diff
@@ -297,6 +298,50 @@ def fix_edf_header(fd):
     print("done")
 
 
+def get_patient_age(header):
+    """Get the age of the patient in days from the header of the edf file.
+
+    Parameters
+    ----------
+    header: Header
+        The EDF header object containing information about the data.
+    """
+    # get the info
+    for header_info, val in zip(HEADER, header, strict=False):
+        field_name = header_info[0]
+        if field_name == "local_patient_identification":
+            lpi = val.split(" ")
+            birthdate = lpi[2]
+        elif field_name == "local_recording_identification":
+            lri = val.split(" ")
+            recdate = lri[1]
+        elif field_name == "startdate_of_recording":
```

Comment:
The start date is defined in two distinct locations. Check the consistency:
- _startdate_ and _starttime_ in header record
- the 2nd subfield of the _local recording identification_  field:
  > - The startdate itself in `dd-MMM-yyyy` format using the English 3-character abbreviations of the month in capitals.